src/client/pmix_client.c

```diff
@@ -574,9 +585,21 @@ static void _putfn(int sd, short args, void *cbdata)
         /* shouldn't be possible */
         goto done;
     }
+#if defined(PMIX_ENABLE_DSTORE) && (PMIX_ENABLE_DSTORE == 1)
+    /* TODO: It is not safe to store data on a client side
+     * There is a possibility to get server/client conflict.
+     * Do nothing here misses PMIx_Get/PMIx_Put flow (w/o PMIx_Commit)
+     */
+    /*
+    if (PMIX_SUCCESS != (rc = pmix_dstore_store(ns->nspace, pmix_globals.myid.rank, kv))) {
+        PMIX_ERROR_LOG(rc);
+    }
+    */
+#else
```

Comment:
@igor-ivanov 
so what's the recent status of this? will you implement dstore as you've mentioned or we will use tiny hash table with local data as we agreed on one of the calls?